paddle/fluid/inference/api/mkldnn_quantizer_config.cc

```diff
@@ -54,6 +54,14 @@ MkldnnQuantizerConfig::MkldnnQuantizerConfig() {
   rules_["matmul"]["Y"] = ScaleAlgo::KL;
   rules_["matmul"]["Out"] = ScaleAlgo::KL;
 
+  rules_["elementwise_add_onednn"]["X"] = ScaleAlgo::KL;
+  rules_["elementwise_add_onednn"]["Y"] = ScaleAlgo::KL;
+  rules_["elementwise_add_onednn"]["Out"] = ScaleAlgo::KL;
+
+  rules_["elementwise_mul_onednn"]["X"] = ScaleAlgo::KL;
+  rules_["elementwise_mul_onednn"]["Y"] = ScaleAlgo::KL;
+  rules_["elementwise_mul_onednn"]["Out"] = ScaleAlgo::KL;
+
```

Comment:
Don't we need `rules_` section for `elementwise_sub_onednn` as well ? What is your opinion?

Reply:
There is no rule for elementwise_sub operator.
I added these rules only for ops which already have rules.

Reply:
I agree elementwise_sub is not quantized yet, so there is no need to add it now.